stat/util.lua

```diff
@@ -57,4 +57,26 @@ function util.gethostname()
   return ffi.string(name_buf)
 end
 
+function util.get_replica_host(peer)
+    local _, host = string.match(peer, '(.+)@(.+)')
+    return host
+end
+
+function util.get_my_replica_host()
```

Comment:
It is intended to return local host:port? It is better to parse box.cfg.listen so.